docs/Templates/SpikeOutcome-Template.md

```diff
@@ -51,4 +51,4 @@ Often based on any open issues/risks Identified. You may state that another spik
 the team to resolve new issues identified (or) indicate that this spike has increased the teams
 confidence in XYZ and should move on.\*
 
-- e.g. Recommendation X: \*\*
+- e.g. Recommendation X: \*\*
```

Comment:
Please avoid including file modifications with non-changes such as this in pull requests, as it is unnecessary and makes the pull harder to review.